runtime/ftplugin/xdefaults.vim

```diff
@@ -13,7 +14,7 @@ set cpo&vim
 
 let b:undo_ftplugin = "setl com< cms< inc< fo<"
 
-setlocal comments=s1:/*,mb:*,ex:*/,:! commentstring& inc&
+setlocal comments=s1:/*,mb:*,ex:*/,:! commentstring=/*\ %s\ */ inc&
```

Comment:
I think the default here was actually incorrect; the correct commentstring is `!\ %s`, see https://github.com/tpope/vim-commentary/issues/85